CHANGELOG.md

```diff
@@ -1,5 +1,26 @@
 # Changelog
 
+## 0.6.1
+
+Bug fixes:
+- Fix flaky `ModelInsight` tests [#395](https://github.com/salesforce/TransmogrifAI/pull/395)
+- Avoid creating `SparseVector`s for LOCO [#377](https://github.com/salesforce/TransmogrifAI/pull/377)
+
+New features / updates:
+- Model combiner [#385](https://github.com/salesforce/TransmogrifAI/pull/399)
+- Added new sample for HousingPrices [#365](https://github.com/salesforce/TransmogrifAI/pull/365)
+- Test to verify that custom metrics appear in model insight metrics [#387](https://github.com/salesforce/TransmogrifAI/pull/387)
+- Add `FeatureDistribution` to `SerializationFormat`s [#383](https://github.com/salesforce/TransmogrifAI/pull/383)
+- Add metadata to `OpStandadrdScaler` to allow for descaling [#378](https://github.com/salesforce/TransmogrifAI/pull/378)
+- Improve json serde error in `evalMetFromJson` [#380](https://github.com/salesforce/TransmogrifAI/pull/380)
+- Track mean & standard deviation of text length as a metric for text feature [#354](https://github.com/salesforce/TransmogrifAI/pull/354)
+- Making model selectors robust to failing models [#372](https://github.com/salesforce/TransmogrifAI/pull/372)
```

Comment:
also there is a PR to fix an issue with models that fail in some CV fold but not all that we should get in before the release here: https://github.com/salesforce/TransmogrifAI/pull/404